cloudbuild.yaml

```diff
@@ -0,0 +1,39 @@
+steps:
+# Import builder if exists.
+# Note: the reason to use bash is to be able to return an exit code 0 even if
+# the docker image doesn't exist yet.
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker pull gcr.io/$PROJECT_ID/yarn_with_rust:node-${_NODE_VERSION} || exit 0
+
+# Build a modified version of the yarn builder, adding rust to the image.
+# Use the previous built image as cache.
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+  - build
+  - --build-arg=NODE_VERSION=${_NODE_VERSION}
+  - -t
+  - gcr.io/$PROJECT_ID/yarn_with_rust:node-${_NODE_VERSION}
+  - --cache-from
+  - gcr.io/$PROJECT_ID/yarn_with_rust:node-${_NODE_VERSION}
+  - -f
+  - cloud_builders/yarn_with_rust/Dockerfile
```

Comment:
If you run this in `dir: cloud_builders/yarn_with_rust` you can drop both of these lines

Reply:
Love it, thanks!